src/functional_data.Rmd

```diff
@@ -0,0 +1,177 @@
+# Functional Data
+
+Functional data provides information about curves varying over a continuum, such as time.
```

Comment:
Done, see next commit. 

** I may not be your target audience** -> You are exactly the target audience!